install/install.sh

```
@@ -0,0 +1,184 @@
+clear
+
+repo=https://github.com/KaraSolar/loveletter.git
+repo_crons=https://github.com/KaraSolar/Rpi_Crons.git
+
+tags=($(git ls-remote --tags $repo | awk -F'/' '{print $NF}'))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)